src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp

```diff
@@ -69,6 +69,20 @@ std::vector<ov::element::Type> prcs = {
     ov::element::u64,
 };
 
+std::vector<ov::element::Type> supported_input_prcs = {
+        ov::element::boolean,
+        ov::element::f16,
+        ov::element::f32,
+        ov::element::i8,
+         // ov::element::i16,
```

Comment:
#10180 adds i16/u16 (along with a fix on the G{U plugin)

Reply:
Maybe will be better to use skip config to disab;e these tests